lib/view_model/after_auth_view_models/profile_view_models/edit_profile_view_model.dart

```diff
@@ -63,26 +61,6 @@ class EditProfilePageViewModel extends BaseModel {
     }
   }
 
-  /// This function is used to convert the image into Base64 format.
-  ///
-  /// **params**:
-  /// * `file`:  Takes the image in format of file.
-  ///
-  /// **returns**:
-  /// * `Future<String>`: image in string format
-  Future<String> convertToBase64(File file) async {
-    try {
-      final List<int> bytes = await file.readAsBytes();
-      String base64String = base64Encode(bytes);
-      base64String = base64String;
-      print(base64String);
-      imageFile = base64String as File?;
-      return base64String;
-    } catch (error) {
-      return '';
-    }
-  }
-
   /// This function remove the selected image.
   ///
```

Comment:
Why we need to remove this ?

Reply:
This function is not being used by the views, in edit_profile, whereas it  is being used in the add_profile segment. And so I have removed it from here and added tests for this function used in add_profile tests